partner_firstname/models/res_partner.py

```diff
@@ -73,7 +74,7 @@ def create(self, vals_list):
             # pylint: disable=W8121
             created_partners |= super(
                 ResPartner, self.with_context(partner_context)
-            ).create(vals_list)
+            ).create(vals)
```

Comment:
```suggestion
            ).create([vals])
```

Reply:
Done!
Since it works either way I'm curious: is there any particular reason to prefer this syntax to the one I proposed?

Reply:
Both ways work, but the old one is deprecated and it's internally "translated" to the new one, so better use the new one directly.